pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -43,7 +43,7 @@ func TestLoadTemplatesByTiers(t *testing.T) {
 				t.Run(tier, func(t *testing.T) {
 					for _, kind := range []string{"code", "dev", "stage"} {
 						t.Run(kind, func(t *testing.T) {
-							if kind == "code" && (tier == "base" || tier == "basedeactivationdisabled" || tier == "team") {
+							if kind == "code" && (tier == "advanced" || tier == "base" || tier == "basedeactivationdisabled" || tier == "team" || tier == "test") {
```

Comment:
wouldn't it be easier to check just:
```suggestion
							if kind == "code" && tier != "basic" {
```

Reply:
I guess new tiers shouldn't have the code namespace either so that's true!